godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/String.kt

```diff
@@ -2,41 +2,51 @@
 
 package godot.core
 
-import godot.gdnative.*
+import godot.gdnative.godot_string
 import kotlinx.cinterop.*
 
 
 //From Godot to Kotlin
-fun String(mem: COpaquePointer): String {
-    val gdString = mem.reinterpret<godot_string>()
-    val s = gdString.pointed.readValue().toKString()
-    godot_string_destroy(gdString)
```

Comment:
But don't we need to destroy it after readValue (which is a copy right?) to not leak memory?

Reply:
I don't know if I'm destroying a String still in use in Godot. For now I prefer to risk a leak rather than a crash.